Lecture4/App.js

```diff
@@ -1,14 +1,41 @@
 import React from "react";
+import { useState } from "react"
 import { createRoot } from "react-dom/client";
-import HeaderComponent from "./HeaderComponent/HeaderComponent";
-import CardComponent from "./CardComponent/CardComponent";
+import HeaderComponent from "./Component/HeaderComponent";
+import SearchComponent from "./Component/SearchComponent";
+import CardComponent from "./Component/CardComponent";
+import data from './data.json'
+
+const BodyComponent = () => {
+  const [filteredData, setFilteredData] = useState(data);
+  return (
+    <>
+      <SearchComponent setFilteredData={setFilteredData} />
+      {filteredData.length != 0 ? <CardContainerComponent filteredData={filteredData} /> : <NoRecord />}
+    </>
+  )
+}
+
+
+const NoRecord = () => (<div style={{ textAlign: "center", color: "white" }}>
```

Comment:
It is good that you're trying inline styling. But don't use it. Not readable at all in large applications.